tests/unit/s2n_mem_usage_test.c

```diff
@@ -22,6 +22,14 @@
     #include <sys/sysctl.h>
     /* clang-format on */
     #include <sys/user.h>
+#elif defined(__OpenBSD__)
+    #undef _POSIX_C_SOURCE
+    /* clang-format off */
+    #include <kvm.h>
+    #include <unistd.h>
+    #include <sys/types.h>
+    #include <sys/sysctl.h>
+    /* clang-format on */
```

Comment:
Also if you want to see this yourself you can install clang-format 15.

This will print the changed file to std out.
```bash
clang-format -style=file ./path/to/file.c
```

To change the file in place:
```bash
clang-format -style=file -i ./path/to/file.c
```

To preview the issues that will be fixed:
```bash
clang-format -style=file --dry-run ./path/to/file.c
```

Reply:
Thanks for that. The latest push builds and passes clang-format-15.